examples/Cloudy/Cloudy_example.jl

```diff
@@ -118,8 +118,8 @@ exp_transform(a::AbstractArray) = exp.(a)
 N_ens = 50 # number of ensemble members
 N_iter = 5 # number of EKI iterations
 # initial parameters: N_ens x N_params
-initial_params = EKI.construct_initial_ensemble(N_ens, priors; rng_seed=6)
-ekiobj = EKI.EKIObj(initial_params, param_names, truth.mean, truth.obs_noise_cov, Δt=1.0)
+initial_params = EKP.construct_initial_ensemble(N_ens, priors; rng_seed=6)
+ekiobj = EKP.EKObj(initial_params, param_names, truth.mean, truth.obs_noise_cov, Δt=1.0)
```

Comment:
Is this missing the new Process argument? I believe there is no default constructor for this argument

Reply:
Yes, thanks! Fixed it. The original implementation of the new `EKObj` had Inversion as the default process, and the omission of the Process argument was a leftover from that. In general I am still not sure whether it would be better to have separate constructors for Inversion EKObjs and Sampler EKObjs, as the optional time step argument is unused (and hence potentially confusing) for EK Samplers...